docs/concepts/how-celestia-works/data-availability-layer.md

```diff
@@ -59,10 +59,10 @@ feasible for resource-limited light nodes. However, in order to validate
 block headers, Celestia light nodes need to download the 4k intermediate
 Merkle roots.
 
-For a block data size of n bytes, this means that every light node must
-download O(n) bytes. Therefore, any improvement in the bandwidth capacity
-of Celestia light nodes has a quadratic effect on the throughput of Celestia's
-DA layer.
+For a block data size of n bytes, this means that every light node must download
+O(sqrt(n)) bytes. Therefore, any improvement in the bandwidth capacity of
```

Comment:
Hey @rootulp in the latest PR merged, we won't need to specify O(sqrt(n)) as it was fixed on the `n bytes` side. if you just remove this bit, I will approve the PR

Reply:
https://github.com/celestiaorg/docs/pull/345 is technically correct but doesn't fix all occurrences (i.e. line 90 of this PR)